frontend/src/components/Home/SidePaneView.js

```diff
@@ -43,6 +50,20 @@ export const SidePaneView = () => {
             dispatch(fetchJobsData(startDate, endDate))
     }, [startDate, endDate, dispatch])
 
+    useEffect(() => {
+        let buildParams = ''
+        if(ciSystem !== '') buildParams += `&ciSystem=${ciSystem}`
```

Comment:
Do we need these empty checks for all the vars here? So of them already seems to be having a non-empty value set [here](https://github.com/cloud-bulldozer/ocp-perf-dashboard/blob/main/frontend/src/store/reducers/InitialData.js#L14-L19).

Reply:
As safety passes an empty value from the URL.